Explorer/Assets/Scripts/Global/Dynamic/Bootstraper.cs

```diff
@@ -194,40 +194,35 @@ private bool DetectAndConfigureLocalSceneDevelopment(RealmLaunchSettings launchS
             deepLinkString = Regex.Replace(Application.absoluteURL, @"(https?)//(.*?)$", @"$1://$2");
 #endif
 
-            if (string.IsNullOrEmpty(deepLinkString)) return false;
+            if (string.IsNullOrEmpty(deepLinkString)) return;
 
-            // Regex to detect different parameters in Uri based on first param after '//' and then separated by '&'
-            var pattern = @"(?<=://|&)[^?&]+=[^&]+";
+            // Regex for detecting different parameters in the deep link string If we want to
+            // add more we just add for example (?=.*newParam=(?<newParamGroupName>[^&?!*]+))?
+            var pattern = @"^(?=.*realm=(?<realm>[^&?!*]+))?(?=.*position=(?<position>[^&?!*]+))?";
             var regex = new Regex(pattern);
-            var matches = regex.Matches(deepLinkString);
+            var regexMatch = regex.Match(deepLinkString);
 
-            if (matches.Count == 0
-                || (!matches[0].Value.Contains("realm=http://")
-                    && !matches[0].Value.Contains("realm=https://")))
-                return false;
+            if (!regexMatch.Success) return;
 
-            string localRealm = matches[0].Value.Replace("realm=", "");
-            launchSettings.SetLocalSceneDevelopmentRealm(localRealm);
+            string realmParam = "realm";
+            if (!regexMatch.Groups[realmParam].Success) return;
 
-            var positionParam = "position=";
-            if (matches.Count > 1)
-            {
-                for (var i = 1; i < matches.Count; i++)
-                {
-                    string param = matches[i].Value;
-
-                    if (param.Contains(positionParam))
-                    {
-                        param = param.Replace(positionParam, "");
+            string realmParamValue = regexMatch.Groups[realmParam].Value;
+            localSceneDevelopment = realmParamValue.Contains("127.0.0.1:");
```

Comment:
mmm maybe we should check for `http` or pass an extra param like `local-dev=true`.
Because for some pc the first eth0 ip that returns could be localhost or maybe its 192.168.x.x (local ip).

Reply:
I thought we were forcing the "127.0.0.1..." on the other side, cool yeah using a specific param is better, I'll update it.

Reply:
added `local-scene` param recognition.

Now using `decentraland://realm=http://127.0.0.1:8000&position=100,100&local-scene=true` specifies the local scene development state.